test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -920,22 +926,6 @@ def test_successful_subsequent_unscheduled_cof_purchase
     assert_successful_response(response)
   end
 
-  def test_missing_field
```

Comment:
[This test was added recently](https://github.com/activemerchant/active_merchant/commit/3803514ec9275844d4ff713d815cd8413ebaefc9) with the intent of testing whether we are correctly passing the error message back from CyberSource. 

I don't believe that the intent was ever to ensure that we always pass through an empty string for the `country`, so I've deleted this test to match the new functionality. The test below still ensures that we are correctly passing back the error message from CyberSource.